draft-packages/modal/KaizenDraft/Modal/Presets/InputEditModal.spec.tsx

```diff
@@ -74,4 +74,25 @@ describe("<InputEditModal />", () => {
     expect(handleSubmit).toHaveBeenCalledTimes(1)
     expect(handleDismiss).toHaveBeenCalledTimes(0)
   })
+
+  it("supports a secondary action when secondary button is pressed", () => {
+    const handleSubmit = jest.fn()
+    const handleDismiss = jest.fn()
+    const handleSecondaryAction = jest.fn()
+
+    const { getByText } = render(
+      <InputEditModalWrapper
+        onSubmit={handleSubmit}
+        onDismiss={handleDismiss}
+        onSecondaryAction={handleSecondaryAction}
+        dismissLabel="Secondary button"
```

Comment:
I think it would be a good idea to create a new label here specific to the secondary action, 

I'm kinda surprised to see that the cross button in the top right isn't using the `dismissLabel` passed in, I think it probably should be.

Either way, having to use the `dismissLabel` prop for the secondary button label is going to be quite confusing.

Reply:
Yeah, makes sense, changes made. This will need a cleanup done afterwards. That'd be a major release, otherwise we'll end up with too many props doing similar things.